spec/features/login_account_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
```

Comment:
I mocked the account requests with the pattern here: https://github.com/pulibrary/orangelight/blob/development/spec/views/account/account_spec.rb#L19.